vllm/platforms/xpu.py

```diff
@@ -67,7 +67,11 @@ def get_device_name(cls, device_id: int = 0) -> str:
 
     @classmethod
     def get_punica_wrapper(cls) -> str:
-        return "vllm.lora.punica_wrapper.punica_gpu.PunicaWrapperGPU"
+        xpu_use_triton_kernel = os.getenv("XPU_USE_TRITON_KERNEL", "0") == "1"
```

Comment:
Maybe we keep this env in our internal code but for upstream, let's just use ipex kernel path.

Reply:
Currently, my internal tests show that the Triton kernel is better than the XPU kernel in torch.compile mode. Keeping this option allows for its use in torch.compile?

Reply:
I prefer name it to `XPU_PUNICA_USE_TRITON`